controllers/controllers.go

```diff
@@ -56,6 +57,9 @@ func AddToManager(ctx *pkgctx.ControllerManagerContext, mgr manager.Manager) err
 	if err := virtualmachinepublishrequest.AddToManager(ctx, mgr); err != nil {
 		return fmt.Errorf("failed to initialize VirtualMachinePublishRequest controller: %w", err)
 	}
+	if err := virtualmachinegrouppublishrequest.AddToManager(ctx, mgr); err != nil {
+		return fmt.Errorf("failed to initialize VirtualMachineGroupPublishRequest controller: %w", err)
+	}
```

Comment:
Is there a dedicated capability for this feature? If not, we should probably guard it with `pkgcfg.FromContext(ctx).Features.VMGroups` (not enabled yet) to avoid breaking the main.